entries/a/adac.de.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Raisin": {
+    "domain": "adac.de",
+    "tfa": [
+      "sms",
+      "call"
+    ],
```

Comment:
Please attach some screenshots of the 2FA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.